share/extender/build_input.yml

```diff
@@ -358,7 +358,7 @@ platforms:
         initialMemory: 33554432
         engineJsLibs:   ["sys", "script", "sound", "render"]
         externalJsLibs: ["glfw"]
-        defines:    ["DM_PLATFORM_HTML5", "GL_ES_VERSION_2_0"]
+        defines:    ["DM_PLATFORM_HTML5"]
```

Comment:
Having it "globally" restricted builds where the version shouldn't be defined (e.g. Rive libraries)